client/push/push_manager.js

```diff
@@ -25,13 +25,40 @@ export default class Push {
   }
 
   /**
+   *  get push permission status
+   */
+  getPushPermissionStatus () {
+    // query push permissions
+    return new Promise(function(resolve, reject) {
+      navigator.permissions.query({name:'push', userVisibleOnly:true})
+      .then(function(permissionStatus) {
+        console.log('push permission state is ', permissionStatus.state);
+        resolve(permissionStatus.state);
+      });
+    });
+  }
+
+  /**
+   * get push subscription with uuid from server
```

Comment:
It looks getting subscription from browser's pushManager, but comment seems getting it from server by uuid.
Please correct it if it is wrong